### DISCUSSION TITLE
Erro no curso  Segurança em APIs ASP.NET com JWT e Bearer Authentication

### DISCUSSION THREAD

**Discussion by Lucas-Reluz** [balta-io/forum#411] [Tira Dúvidas] [answered]
Estou tendo um problema ao rodar o comando dotnet ef migrations add v1 Ele retorna o seguinte erro

![image](https://github.com/balta-io/forum/assets/88803842/8686eb3d-de9c-41ab-916e-bbc63388b4be)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Lucas-Reluz, como vai?
Este erro ocorre por que seu DbContext está em um assembly diferente do qual você está executando sua Migration!

Você pode resolver utilizando o `MigrationsAssembly` e apontando para o projeto que vai conter sua migração (Acredito eu que seja sua API).
```sharp
builder.Services.AddDbContext<AppDbContext>(x =>
            x.UseSqlServer(
                Configuration.Database.ConnectionString,
                b => b.MigrationsAssembly("Namespace.Da.Sua.Api")));
```
